.circleci/config.yml

```diff
@@ -211,7 +211,6 @@ workflows:
             - unit_test
             - browserstack_acceptance_test
             - headless_acceptance_test
-            - translation_test
```

Comment:
I think in the future we might run the portion of translation_test that verifies the .pot file to run on all commits. We could probably use the [add-and-commit github action](https://github.com/EndBug/add-and-commit) that Vaccine is using for their third party licenses generation

Reply:
This PR does still run the translation_test. It's just doesn't block the deployment if it fails